ocaml/typing/untypeast.ml

```diff
@@ -452,6 +452,11 @@ let comprehension sub comp_type comp =
   in
   Jane_syntax.Comprehensions.expr_of ~attrs:[] (comp_type (comprehension comp))
 
+let label = function
```

Comment:
Once you have `Position`, this won't work be quite so easy.

Reply:
I think I'll have to convert all Positions to Labelled, then handle adding the constraint of extension separately as is done in #1579